docs/user-guide/rules.md

```diff
@@ -15,87 +19,107 @@ You can avoid errors with these `no` rules.
-- [`declaration-block-no-duplicate-properties`](../../lib/rules/declaration-block-no-duplicate-properties/README.md): Disallow duplicate properties within declaration blocks (Autofixable) (Ⓡ & Ⓢ).
-- [`font-family-no-duplicate-names`](../../lib/rules/font-family-no-duplicate-names/README.md): Disallow duplicate names within font families (Ⓡ & Ⓢ).
-- [`keyframe-block-no-duplicate-selectors`](../../lib/rules/keyframe-block-no-duplicate-selectors/README.md): Disallow duplicate selectors within keyframe blocks (Ⓡ & Ⓢ).
-- [`no-duplicate-at-import-rules`](../../lib/rules/no-duplicate-at-import-rules/README.md): Disallow duplicate `@import` rules (Ⓡ & Ⓢ).
-- [`no-duplicate-selectors`](../../lib/rules/no-duplicate-selectors/README.md): Disallow duplicate selectors (Ⓡ & Ⓢ).
+| Rule                                                                                                                                                                                                 | ⭐️ | 💅  | 🔧  |
+| ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :-: | :-: | :-: |
+| [`declaration-block-no-duplicate-custom-properties`](../../lib/rules/declaration-block-no-duplicate-custom-properties/README.md)<br/>Disallow duplicate custom properties within declaration blocks. | ✅  | ✅  |     |
+| [`declaration-block-no-duplicate-properties`](../../lib/rules/declaration-block-no-duplicate-properties/README.md)<br/>Disallow duplicate properties within declaration blocks.                      | ✅  | ✅  | ✅  |
+| [`font-family-no-duplicate-names`](../../lib/rules/font-family-no-duplicate-names/README.md)<br/>Disallow duplicate names within font families.                                                      | ✅  | ✅  | ✅  |
+| [`keyframe-block-no-duplicate-selectors`](../../lib/rules/keyframe-block-no-duplicate-selectors/README.md)<br/>Disallow duplicate selectors within keyframe blocks.                                  | ✅  | ✅  |     |
+| [`no-duplicate-at-import-rules`](../../lib/rules/no-duplicate-at-import-rules/README.md)<br/>Disallow duplicate `@import` rules.                                                                     | ✅  | ✅  |     |
+| [`no-duplicate-selectors`](../../lib/rules/no-duplicate-selectors/README.md)<br/>Disallow duplicate selectors.                                                                                       | ✅  | ✅  |     |
 
 ### Empty
 
 Disallow empty things with these `no-empty` rules.
 
-- [`block-no-empty`](../../lib/rules/block-no-empty/README.md): Disallow empty blocks (Ⓡ & Ⓢ).
-- [`comment-no-empty`](../../lib/rules/comment-no-empty/README.md): Disallow empty comments (Ⓡ & Ⓢ).
-- [`no-empty-source`](../../lib/rules/no-empty-source/README.md): Disallow empty sources (Ⓡ & Ⓢ).
+| Rule                                                                                          | ⭐️ | 💅  | 🔧  |
```

Comment:
too bad we can't add a tooltip (title attribute) on the icons

Reply:
I don't think Markdown supports the `title` attribute. But we can add anything on the website. I'll try it on stylelint.io

Reply:
See https://github.com/stylelint/stylelint.io/pull/328/commits/91ef25bda66435f08e33ad41473e6a23d47bb5e1